### ISSUE TITLE
[Bug]: PHP Deprecated in VarDumper

### ISSUE THREAD

**Issue by gander** [beyondcode/tinkerwell-community#548]
### Operating system

LinuxMint 20

### Laravel Version

8.82.0 internal/default

### PHP Version

8.1.2

### Project Location

Local

### Bug description

Exceptation:
```
true
=> true
```
Current result:

```
PHP Deprecated:  Return type of _PhpScoperc223a629f245\Symfony\Component\VarDumper\Cloner\Data::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///tmp/.mount_Tinker2vpBHK/resources/tinkerwell/tinker.phar/vendor/symfony/var-dumper/Cloner/Data.php on line 99
PHP Deprecated:  Return type of _PhpScoperc223a629f245\Symfony\Component\VarDumper\Cloner\Data::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///tmp/.mount_Tinker2vpBHK/resources/tinkerwell/tinker.phar/vendor/symfony/var-dumper/Cloner/Data.php on line 103
PHP Deprecated:  Return type of _PhpScoperc223a629f245\Symfony\Component\VarDumper\Cloner\Data::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///tmp/.mount_Tinker2vpBHK/resources/tinkerwell/tinker.phar/vendor/symfony/var-dumper/Cloner/Data.php on line 107
PHP Deprecated:  Return type of _PhpScoperc223a629f245\Symfony\Component\VarDumper\Cloner\Data::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///tmp/.mount_Tinker2vpBHK/resources/tinkerwell/tinker.phar/vendor/symfony/var-dumper/Cloner/Data.php on line 111
PHP Deprecated:  Return type of _PhpScoperc223a629f245\Symfony\Component\VarDumper\Cloner\Data::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///tmp/.mount_Tinker2vpBHK/resources/tinkerwell/tinker.phar/vendor/symfony/var-dumper/Cloner/Data.php on line 76
PHP Deprecated:  Return type of _PhpScoperc223a629f245\Symfony\Component\VarDumper\Cloner\Data::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///tmp/.mount_Tinker2vpBHK/resources/tinkerwell/tinker.phar/vendor/symfony/var-dumper/Cloner/Data.php on line 80
true
=> true
```

### Steps to reproduce

```php
dump(true);
```

**Comment by mpociot** [comment]
Thanks for reporting this. Looks like we introduced this bug in the latest release. We'll fix this in the version that'll be released next week along with the Laravel 9 update

**Comment by sschlein** [comment]
This has been fixed (tested with Tinkerwell 2.24)
